*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 12:23:45 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: 9a0df6d52767
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:42779 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x558a0ab0f1d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x558a0ab0f1d0 pid: 11 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 11)
spawned uWSGI worker 1 (pid: 13, cores: 20)
Mon Aug  8 20:23:45 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 15, cores: 20)
spawned uWSGI http 1 (pid: 23)
Mon Aug  8 20:23:45 2022 - mem-collector thread started for worker 2
*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 12:30:41 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: b3935028b64b
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:44655 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x55e446f371d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x55e446f371d0 pid: 10 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 10)
spawned uWSGI worker 1 (pid: 12, cores: 20)
Mon Aug  8 20:30:41 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 14, cores: 20)
spawned uWSGI http 1 (pid: 21)
Mon Aug  8 20:30:41 2022 - mem-collector thread started for worker 2
*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 12:31:13 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: b3935028b64b
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:33993 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x5606898121d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5606898121d0 pid: 10 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 10)
spawned uWSGI worker 1 (pid: 12, cores: 20)
Mon Aug  8 20:31:13 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 14, cores: 20)
spawned uWSGI http 1 (pid: 21)
Mon Aug  8 20:31:13 2022 - mem-collector thread started for worker 2
*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 12:55:03 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: 2281b340e16c
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:33691 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x5653a07621d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5653a07621d0 pid: 15 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15)
spawned uWSGI worker 1 (pid: 17, cores: 20)
Mon Aug  8 20:55:03 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 19, cores: 20)
Mon Aug  8 20:55:03 2022 - mem-collector thread started for worker 2
spawned uWSGI http 1 (pid: 25)
INFO:apscheduler.threadpool:Started thread pool with 0 core threads and 20 maximum threads
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:57:13.536532)" to job store "default"
INFO:apscheduler.scheduler:Added job "clear_task (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 20:56:43.536686)" to job store "default"
INFO:apscheduler.scheduler:Added job "clean_task_log (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 20:56:43.536760)" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:root:[08/Aug/2022 20:56:43] POST /account/?action=login execute_time 0.289507
[pid: 17|app: 0|req: 1/1] 172.22.0.1 () {32 vars in 406 bytes} [Mon Aug  8 20:56:43 2022] POST /account/?action=login => generated 112 bytes in 297 msecs (HTTP/1.1 200) 5 headers in 171 bytes (1 switches on core 0)
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:57:13.536532)" (scheduled at 2022-08-08 20:57:13.536532)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:57:43.536532)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:57:43.536532)" (scheduled at 2022-08-08 20:57:43.536532)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:58:13.536532)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:58:13.536532)" (scheduled at 2022-08-08 20:58:13.536532)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:58:43.536532)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:58:43.536532)" (scheduled at 2022-08-08 20:58:43.536532)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:59:13.536532)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:59:13.536532)" (scheduled at 2022-08-08 20:59:13.536532)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 20:59:43.536532)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 12:59:58 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: b06c22da1171
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:35523 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x5643c10c51d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5643c10c51d0 pid: 14 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 14)
spawned uWSGI worker 1 (pid: 16, cores: 20)
Mon Aug  8 20:59:58 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 18, cores: 20)
Mon Aug  8 20:59:58 2022 - mem-collector thread started for worker 2
spawned uWSGI http 1 (pid: 24)
INFO:apscheduler.threadpool:Started thread pool with 0 core threads and 20 maximum threads
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:00:39.097630)" to job store "default"
INFO:apscheduler.scheduler:Added job "clear_task (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 21:00:09.097803)" to job store "default"
INFO:apscheduler.scheduler:Added job "clean_task_log (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 21:00:09.097968)" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:root:[08/Aug/2022 21:00:08] POST /account/?action=login execute_time 0.322095
[pid: 16|app: 0|req: 1/1] 172.22.0.1 () {32 vars in 405 bytes} [Mon Aug  8 21:00:08 2022] POST /account/?action=login => generated 93 bytes in 331 msecs (HTTP/1.1 200) 5 headers in 170 bytes (1 switches on core 2)
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:00:39.097630)" (scheduled at 2022-08-08 21:00:39.097630)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:01:09.097630)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:01:09.097630)" (scheduled at 2022-08-08 21:01:09.097630)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:01:39.097630)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
*** Starting uWSGI 2.0.20 (64bit) on [Mon Aug  8 13:01:52 2022] ***
compiled with version: 8.3.0 on 08 August 2022 04:32:36
os: Linux-5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021
nodename: 27e87e2986a7
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /bioinformatics-analysis
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /bioinformatics-analysis
your memory page size is 4096 bytes
 *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers *** 
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: enabled
uWSGI http bound on 0.0.0.0:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:42683 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.6.13 (default, Apr 10 2021, 16:30:36)  [GCC 8.3.0]
Python main interpreter initialized at 0x558ce42fb1d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 814752 bytes (795 KB) for 40 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x558ce42fb1d0 pid: 15 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15)
spawned uWSGI worker 1 (pid: 17, cores: 20)
Mon Aug  8 21:01:53 2022 - mem-collector thread started for worker 1
spawned uWSGI worker 2 (pid: 19, cores: 20)
spawned uWSGI http 1 (pid: 26)
Mon Aug  8 21:01:53 2022 - mem-collector thread started for worker 2
INFO:apscheduler.threadpool:Started thread pool with 0 core threads and 20 maximum threads
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:02:31.352040)" to job store "default"
INFO:apscheduler.scheduler:Added job "clear_task (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 21:02:01.352167)" to job store "default"
INFO:apscheduler.scheduler:Added job "clean_task_log (trigger: interval[30 days, 0:00:00], next run at: 2022-09-07 21:02:01.352240)" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:root:[08/Aug/2022 21:02:01] POST /account/?action=login execute_time 0.320432
[pid: 17|app: 0|req: 1/1] 172.22.0.1 () {32 vars in 405 bytes} [Mon Aug  8 21:02:01 2022] POST /account/?action=login => generated 217 bytes in 329 msecs (HTTP/1.1 200) 5 headers in 171 bytes (1 switches on core 0)
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:02:31.352040)" (scheduled at 2022-08-08 21:02:31.352040)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:03:01.352040)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:03:01.352040)" (scheduled at 2022-08-08 21:03:01.352040)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:03:31.352040)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
INFO:apscheduler.scheduler:Running job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:03:31.352040)" (scheduled at 2022-08-08 21:03:31.352040)
ERROR:apscheduler.scheduler:Job "run_task (trigger: interval[0:00:30], next run at: 2022-08-08 21:04:01.352040)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "./bioinformatics/urls.py", line 46, in run_task
    max_task = Config.objects.filter(name="max_task").first().value
AttributeError: 'NoneType' object has no attribute 'value'
